lib/htmlValidator.js

```diff
@@ -186,11 +198,28 @@ module.exports = function (app, callback) {
         return validationMessage
       }
 
-      if (req.headers[headerException]) {
-        res.set(headerException, true)
+      function validatorExceptions () {
+        if (foundModelException) {
+          return true
+        }
+
+        headerException = [].concat(headerException)
+
+        headerException.forEach(function (exception) {
+          // check the request header
+          if (req.headers[exception.toLowerCase()]) {
+            res.set(exception.toLowerCase(), true)
+            return true
```

Comment:
the reason the found variable was used is this won't work

you're returning inside the forEach callbac, not the parent function